clients/unshaded/src/main/java/tachyon/client/table/AbstractTachyonRawTables.java

```diff
@@ -30,10 +29,10 @@
  */
 @PublicApi
 public abstract class AbstractTachyonRawTables implements TachyonRawTablesCore {
-  protected RawTablesContext mContext;
+  protected RawTableContext mContext;
```

Comment:
Why do we have RawTables(Core, Context, ...) and RawTable(Client, ClientPool, ...)? Where does one begin and the other one end? Could we just use RawTable everywhere for consistency?